src/buffer/out/textBuffer.cpp

```diff
@@ -2920,16 +2920,13 @@ void TextBuffer::ScrollMarks(const int delta)
     _trimMarksOutsideBuffer();
 }
 
-void TextBuffer::AddMark(const ScrollMark& m, const bool activeMark)
+void TextBuffer::StartPromptMark(const ScrollMark& m)
 {
-    if (activeMark)
-    {
-        _marks.push_back(m);
-    }
-    else
-    {
-        _marks.insert(_marks.begin(), m);
-    }
+    _marks.push_back(m);
+}
+void TextBuffer::AddMark(const ScrollMark& m)
+{

```

Comment:
I still don't understand why "prompt mark" is "end of the entire list of marks" and "add mark" is the "beginning". I was hoping to find some clarity... but can you please just explain the guiding principle here?

Reply:
Reading other comments, I understand that "AddMark" is "add a mark that is not the active mark" and "StartPromptMark" is "add a mark that IS the active mark" :smile: